jetty-util/src/main/java/org/eclipse/jetty/util/thread/strategy/AdaptiveExecutionStrategy.java

```diff
@@ -462,7 +470,10 @@ private void execute(Runnable task)
     {
         try
         {
-            _executor.execute(task);
+            if (_useVirtualThreads)
+                VirtualThreads.startVirtualThread(task);
+            else
+                _executor.execute(task);
```

Comment:
Wow, that's pretty simple...
Makes me wonder if we could do it all with just a change to the ThreadPool?   Hmmm but then we'd have virtual threads being selectors.... so nah!